.github/workflows/js-css-coding-standards.yml

```diff
@@ -0,0 +1,43 @@
+name: JS and CSS Coding Standards
+
+on:
+  push:
+    branches: [ trunk, develop ]
+  pull_request:
+    branches: [ trunk, develop ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Adding Node.js setup to run JS and CSS linters
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
```

Comment:
Any reason why you're using `@v2` instead of `@v3` which is the latest?

Reply:
Updated!